python/torch_mlir/dialects/torch/importer/jit_ir/csrc/torch_to_mlir_utils.cpp

```diff
@@ -21,6 +21,7 @@
 #include "mlir-c/Diagnostics.h"
 #include "torch-mlir-c/TorchOps.h"
 #include "torch-mlir-c/TorchTypes.h"
+#include "torch-mlir/Dialect/Torch/IR/TorchTypes.h"
```

Comment:
This completely breaks the layering of the Python importer code. We should not be importing `torch-mlir/`, only `torch-mlir-c/`.